js/src/tab.js

```diff
@@ -43,9 +43,8 @@ const CLASS_NAME_FADE = 'fade'
 const CLASS_NAME_SHOW = 'show'
 
 const SELECTOR_DROPDOWN = '.dropdown'
-const SELECTOR_NAV_LIST_GROUP = '.nav, .list-group'
+const SELECTOR_NAV_PARENT = '.nav, .list-group, .btn-toolbar'
 const SELECTOR_ACTIVE = '.active'
```

Comment:
I added one more class `.btn-toolbar` here to give the support of button toolbar in navs. But I was thinking that there may be some unique selector here as well that selects the parent of the navs in every case. Like we can use `data-toggle="something"`. I am not sure but can you please suggest @XhmikosR, if it is a good idea. 🙂